Lil-Peep/src/snippets/body-collection.liquid

```diff
@@ -42,77 +42,79 @@
   </header>
 
   <div class="collection-container">
-    {% for product in collection.products %}
-      <div class="product-card-category">
-        <div class="products-slider">
-          <div class="front-label">
-          {% assign date_created = product.created_at | date:'%s' %}
-          {% assign     date_now = 'now' | date:'%s' %}
-          {% assign   difference = date_now | minus: date_created %}
-
-          {% if difference < 172800 %}
-            <p class="new-product">NEW</p>
-          {% else %}
-            <p class="not-new-product">NOT NEW</p>
-          {% endif %}
-          </div>
-          <div class="product-image">
-            {% if product.featured_image != blank %}
-              <a href="{{ product.url | within: collection }}">
-                {{ product.featured_image.src | img_url: '480x480' | img_tag: product.featured_image.alt }}
-              </a>
+    <div>
+      {% for product in collection.products %}
+        <div class="product-card-category">
+          <div class="products-slider">
+            <div class="front-label">
+            {% assign date_created = product.created_at | date:'%s' %}
+            {% assign     date_now = 'now' | date:'%s' %}
+            {% assign   difference = date_now | minus: date_created %}
+
+            {% if difference < 172800 %}
+              <p class="new-product">NEW</p>
+            {% else %}
+              <p class="not-new-product">NOT NEW</p>
             {% endif %}
+            </div>
+            <div class="product-image">
+              {% if product.featured_image != blank %}
+                <a href="{{ product.url | within: collection }}">
+                  {{ product.featured_image.src | img_url: '480x480' | img_tag: product.featured_image.alt }}
+                </a>
+              {% endif %}
+            </div>
           </div>
-        </div>
-        <div class="product-data">
-          <p>
-            <a href="{{ product.url | within: collection }}" class="product-title">{{ product.title }}</a>
-          </p>
-          <div class="price">
-            <p class="price">
+          <div class="product-data">
+            <p>
+              <a href="{{ product.url | within: collection }}" class="product-title">{{ product.title }}</a>
+            </p>
+            <div class="price">
+              <p class="price">
```

Comment:
yep, it's just a color specification.

Reply:
🤔🤔🤔🤔🤔🤔🤔🤔🤔
It could be just a difference, but remember to keep the semantic, not only of your HTML tags, also with class names 🤓 , I'd suggest to choose `price-container` to the `div` and `price`  to the `p`